multi/imxrt-multi/imxrt-multi.c

```diff
@@ -419,9 +439,6 @@ static int createDevFiles(void)
 	if (mkFile(&dir, id_trng, "trng", multi_port) < 0) {
 		return -1;
 	}
-#endif
```

Comment:
`id_trng` msg is only handled if `#ifndef __CPU_IMXRT117X`, so there is probably no point in creating random/trng devices

Reply:
It is still behind `TRNG` ifdef so there should be no devices, but I added `#ifndef __CPU_IMXRT117X` to be sure.